interceptors/logging/logging.go

```diff
@@ -55,6 +55,10 @@ func DefaultDeciderMethod(_ string, _ error) bool {
 	return true
 }
 
+// PostRequestLoggingDecider is a user provided function for deciding whether the Request Logging should be configured.
+// Request Logging is enabled if the value of the function evaluates to true.
+type PostRequestLoggingDecider func(ctx context.Context, fullMethodName string, servingObject interface{}) bool
```

Comment:
Do you mean PreRequest? Also "Request logging" is whole thing. I think you mean "start log entry", right?

Reply:
Yeah, I didn't  change the naming after we had the discussion, will change it to PreRequest!